### ISSUE TITLE
Drawer: Sticky bottom nav doesn't behave properly

### ISSUE THREAD

**Issue by csechuan** [saadeghi/daisyui#1520]
Sticky bottom nav doesn't behave properly when open drawer.

Reproduce:
Try scroll down a bit on main drawer content before open the drawer. You will notice the bottom nav will move upward as much as you scroll down in the main.  

[https://play.tailwindcss.com/S9zsrSPbBc](https://play.tailwindcss.com/S9zsrSPbBc)

**Comment by cappe** [comment]
I noticed this too this morning. It's the following CSS style in TailwindCSS' @components layer:

```
.drawer-toggle:checked ~ .drawer-content,
.drawer.drawer-end > .drawer-toggle:checked ~ .drawer-content {
  transform:  translate(var(--tw-translate-x),var(--tw-translate-y))
              rotate(var(--tw-rotate))
              skewX(var(--tw-skew-x)) 
              skewY(var(--tw-skew-y))
              scaleX(var(--tw-scale-x))
              scaleY(var(--tw-scale-y));
}
```

I could not figure out the fix (with limited time) so ended up overriding it with `transform: none;` for now.

**Comment by saadeghi** [comment]
Ok I found the reason: having a `fixed` element inside a `transform`ed element would make it fixed to the transformed element instead of the closest `relative` element. I didn't know that before today.  

Read more:
- https://stackoverflow.com/questions/15194313/transform3d-not-working-with-position-fixed-children/15256339#15256339
- http://meyerweb.com/eric/thoughts/2011/09/12/un-fixing-fixed-elements-with-css-transforms/

I really liked that little transition of the `drawer-content` when `drawer` opens but maybe I should remove that if it is causing issues like this 🥲

**Comment by cappe** [comment]
@saadeghi TIL, thanks for providing more details! I really liked the transition too, it made the touch & feel pretty nice.

I'll try to find some time and motivation to dig into it.
